src/aktualizr_info/main.cc

```diff
@@ -46,7 +46,9 @@ int main(int argc, char **argv) {
     ("delegation",  "Outputs metadata of image repo targets' delegations")
     ("director-root",  "Outputs root.json from director repo")
     ("director-target",  "Outputs targets.json from director repo")
-    ("allow-migrate", "Opens database in read/write mode to make possible to migrate database if needed");
+    ("allow-migrate", "Opens database in read/write mode to make possible to migrate database if needed")
+    ("images-snapshot", "Outputs snapshot.json from images repo")
+    ("images-timestamp", "Outputs timestamp.json from images repo");
```

Comment:
Very small complaint: could you move these two items between "images-root" and "images-target"? The ideal order would be: root, timestamp, snapshot, targets. That more or less matches the order they are downloaded by libaktualizr and the order in which they sign each other.

Reply:
Of cause. I moved them.